middle_end/flambda/un_anf.ml

```diff
@@ -161,38 +161,38 @@ let make_var_info (clam : Clambda.ulambda) : var_info =
           captured_variables
       in
       let environment_vars =
-      List.fold_left (fun (
+      List.fold_left (fun environment_vars (
         { Clambda. label; arity; params; return; body; dbg; env; } as clos) ->
-          let environment_vars =
```

Comment:
You probably need to keep this let, it's the only part of the code that directly adds a variable to the `environment_vars` set so you need to make sure you do not discard the result.